README.md

```diff
@@ -1,16 +1,66 @@
-# errors Integration Pack
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Configuration
-TODO: Describe configuration
+# Errors Pack
 
+# Introduction
+This pack holds all of the contains all actions and workflows needed for finding and formating
+StackStorm errors
 
-# Sensors
+# Quick Start
 
-## Example Sensor
-TODO: Describe sensor
+**Steps**
 
+1. Install the pack
 
-# Actions
+    ``` shell
+    st2 pack install https://github.com/EncoreTechnologies/stackstorm-errors.git
+    ```
 
-## example
-TODO: Describe action
+2. Execute an action (example: Build an execution tree)
+
+    ``` shell
+    st2 run errors.build_execution_tree st2_exe_id="testID"
+    ```
+
+# Usage
+
+## Actions
+
+| Action | Description |
+|--------|-------------|
+| build_execution_tree | Builds an execution tree of the given task |
+| get_formatted_error | Finds an error in the given task and formats the result into an HTML tagged output |
+
+### Action Example - errors.build_execution_tree
+
+`errors.build_execution_tree` builds an execution tree of a given StackStorm task:
```

Comment:
For all of these give a sample output so the user knows what to expect and how to handle it in their workflows